templates/common/_base/units/ovs-notification.yaml

```diff
@@ -0,0 +1,21 @@
+name: ovs-notification.service
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Creates a file to let the OVS pod know OVS is running on systemd.
+  # Don't write the notification before OVS starts
```

Comment:
Why not? The goal of the service is to tell the sdn-ovs pod that system OVS is enabled. If this service is running at all, then system OVS is enabled, so it doesn't matter if this runs before or after OVS starts

Reply:
Is the goal to have a `hostPath` `type File` constraint in [ovn-kubernetes/006-ovs-node.yaml ](https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/006-ovs-node.yaml) to keep ovs-node from starting until it sees this file?

Reply:
@danwinship yes I guess that's reasonable.
@mccv1r0 yes